src/rpc/daemon_handler.cpp

```diff
@@ -0,0 +1,726 @@
+      amt.amount = amount;
+      amt.outputs = ofa;
+
+      res.amounts_with_outputs.push_back(amt);
+    }
+
+    res.status = Message::STATUS_OK;
+  }
+
+  void DaemonHandler::handle(SendRawTx::Request& req, SendRawTx::Response& res)
+  {
+    auto tx_blob = cryptonote::tx_to_blob(req.tx);
+
+    cryptonote_connection_context fake_context = AUTO_VAL_INIT(fake_context);
+    tx_verification_context tvc = AUTO_VAL_INIT(tvc);
+
+    if(!m_core.handle_incoming_tx(tx_blob, tvc, false, false) || tvc.m_verifivation_failed)
+    {
+      if (tvc.m_verifivation_failed)
+      {
+        LOG_PRINT_L0("[on_send_raw_tx]: tx verification failed");
+      }
+      else
+      {
+        LOG_PRINT_L0("[on_send_raw_tx]: Failed to process tx");
+      }
+      res.status = Message::STATUS_FAILED;
+
+      //TODO: these should be mutually exclusive, as encountering one
+      //      cancels the verification process.  Need to confirm this.
+      if (tvc.m_low_mixin)
+        res.error_details = "mixin too low";
+      else if (tvc.m_double_spend)
+        res.error_details = "double spend";
+      else if (tvc.m_invalid_input)
+        res.error_details = "invalid input";
+      else if (tvc.m_invalid_output)
+        res.error_details = "invalid output";
+      else if (tvc.m_too_big)
+        res.error_details = "too big";
+      else if (tvc.m_overspend)
+        res.error_details = "overspend";
+      else if (tvc.m_fee_too_low)
+        res.error_details = "fee too low";
+      else if (tvc.m_not_rct)
+        res.error_details = "tx is not ringct";
+      else
+        res.error_details = "an unknown issue was found with the transaction";
+
+      return;
+    }
+
+    if(!tvc.m_should_be_relayed || !req.relay)
+    {
+      LOG_PRINT_L0("[on_send_raw_tx]: tx accepted, but not relayed");
+      res.error_details = "Not relayed";
+      res.relayed = false;
+      res.status = Message::STATUS_OK;
+
+      return;
+    }
+
+    NOTIFY_NEW_TRANSACTIONS::request r;
+    r.txs.push_back(tx_blob);
+    m_core.get_protocol()->relay_transactions(r, fake_context);
+
+    //TODO: make sure that tx has reached other nodes here, probably wait to receive reflections from other nodes
+    res.status = Message::STATUS_OK;
+    res.relayed = true;
+
+    return;
+  }
+
+  void DaemonHandler::handle(StartMining::Request& req, StartMining::Response& res)
+  {
+  }
+
+  void DaemonHandler::handle(GetInfo::Request& req, GetInfo::Response& res)
+  {
+    res.height = m_core.get_current_blockchain_height();
+
+    res.target_height = m_core.get_target_blockchain_height();
+
+    auto& chain = m_core.get_blockchain_storage();
+
+    res.difficulty = chain.get_difficulty_for_next_block();
+
+    res.target = chain.get_difficulty_target();
+
+    res.tx_count = chain.get_total_transactions() - res.height; //without coinbase
+
+    res.tx_pool_size = m_core.get_pool_transactions_count();
+
+    res.alt_blocks_count = chain.get_alternative_blocks_count();
+
```

Comment:
I try to keep separate thoughts separated by a line.  Style preference I suppose.